http_service/tests/test_bug_classification.py

```diff
@@ -159,6 +159,41 @@ def do_request():
     }
 
 
+def test_for_missing_bugs(client, responses):
+    bugs_list = [1602463, 1619699]
+    missing_bugs_list = [1598744, 1615281, 1566486]
+    # merge the two list
+    bug_ids = [*bugs_list, *missing_bugs_list]
+
+    change_time = str(time.time())
+
+    # A call to Buzgilla will return bugs for
+    # IDs in bugs_list only
+    responses.add(
+        responses.GET,
+        "https://bugzilla.mozilla.org/rest/bug?id=1566486,1598744,1602463,1615281,1619699&include_fields=id&include_fields=last_change_time",
+        status=200,
+        json={
+            "bugs": [
+                {"id": bug_id, "last_change_time": change_time} for bug_id in bugs_list
+            ],
+        },
+    )
+
+    rv = client.post(
+        "/component/predict/batch",
+        data=json.dumps({"bugs": bug_ids}),
+        headers={API_TOKEN: "test"},
+    )
+    assert rv.status_code == 202
+    bugs = retrieve_compressed_reponse(rv)["bugs"]
+
+    assert len(bug_ids) == len(bugs.keys()), "All the queried bug IDs must be returned"
```

Comment:
This assertion guarantees the same number of bugs, but does not guarantee that all queried bugs are returned. I would do something like `bugs.keys() == all_bug_ids` (considering `all_bug_ids` is a set).

Reply:
well noted sir